workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectDispatcherSystem.cs

```diff
@@ -9,31 +10,54 @@ namespace Improbable.Gdk.Core.GameObjectRepresentation
     ///     Gathers incoming dispatcher ops and invokes callbacks on relevant GameObjects.
     /// </summary>
     [DisableAutoCreation]
+    [AlwaysUpdateSystem]
```

Comment:
Nice!

Reply:
Is this added because of RunDispatchers? I think it would be nicer to split them up to separate systems in that case... RunDispatchers looks out of place now.

Or, the other way around: the dispatcher system should not be dealing with added / removed entities.

Reply:
I do agree! However the dispatchers use dictionaries that are populated when entities get added. So we either have a static class with this dictioniaries (which I don't like) or all the setup and running logic in one system. Or: put it as a component on the entity that might also be possible, though not sure how much implementation changes you want to do before m1

Reply:
I see, the last option is my favorite. Can you create a ticket with the requirements and the options you suggested for the M2 milestone please, since you have the fresh context on what may need to be / can be done?

Reply:
sure, will do that.

Reply:
UTY-1159